testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -354,6 +354,28 @@ public void testEscapableKeyPut() throws IOException {
             "{\"description\":\"Request invalid: payload must not be empty.\",\"code\":422}");
   }
 
+  @Test
+  public void testTtlPut() throws IOException {
+    JsonNode obj1 = OBJECT_MAPPER.readTree("{ \"delete this\": \"in ten seconds\" }");
+    JsonNode obj2 = OBJECT_MAPPER.readTree("{ \"do not delete\": \"this\", \"a\": \"b\" }");
+    JsonNode subObj2 = OBJECT_MAPPER.readTree("{ \"delete this as well\": \"in ten seconds\" }");
+    RestUtils.put(authToken, collectionPath + "/1?ttl=10", obj1.toString(), 200);
```

Comment:
hmm is TTL in seconds? can we decrease to `1` and sleep only one second as well, no need to wait this long?